NobelLaureateDetails/Pages/Index.cshtml.cs

```diff
@@ -14,31 +14,37 @@ public JsonResult OnGet()
         {
             List<Laureate> filteredLaureates = new List<Laureate>();
 
-            using (WebClient webClient = new WebClient())
-            {
-                string PhysicsNinetyEight = webClient.DownloadString("http://api.nobelprize.org/v1/prize.json?category=physics&year=1998");
-                Prizes.Laureates Laureates = Prizes.Laureates.FromJson(PhysicsNinetyEight);
+            string physicsNinetyEightlaureates = getJsonFromUrl("http://api.nobelprize.org/v1/prize.json?category=physics&year=1998");
+            Prizes.Laureates Laureates = Prizes.Laureates.FromJson(physicsNinetyEightlaureates);
 
-                List<long> filteredIds = new List<long>();
+            List<long> laureatesIds = new List<long>();
```

Comment:
This is good... laureatesIds is more descriptive than filteredIds.  What is being filtered?  I would probably go with a singular form of laureate, and plural of Ids, though... something like laureateIDs would work.